server_tests/test/clientTests.js

```diff
@@ -2238,12 +2613,15 @@ describe("RUST API TEST SUITE", function() {
             var expectedShutdownDelay = 3000;
             var startTime = Date.now();
             testClient.bublShutdown(sessionId, expectedShutdownDelay)
-            .then( Comparison.catchExceptions(done, function(res) {
+            .then( function onSuccess (res) {
                 Comparison.bublShutdownOutput(res);
                 var endTime = Date.now();
                 Comparison.shutdownDelay(startTime, endTime, expectedShutdownDelay);
                 done();
-            }));
+            }, function onError (err) {
```

Comment:
Rather than repeat this code everywhere, why not declare the function once at the top and re-use it?